ansible/install_heat.yaml

```diff
@@ -26,66 +26,34 @@
     with_items:
       - heat-operator
 
-  - name: delete heat-operator resources if they exist
-    shell: |
-      oc delete -f {{ base_path }}/github.com/openstack-k8s-operators/heat-operator/deploy/{{ item }}
-    args:
-      chdir: "{{ base_path }}"
-    environment:
-      PATH: "/usr/local/bin:{{ ansible_env.PATH }}"
-      KUBECONFIG: "{{base_path}}/dev-scripts/ocp/{{ ocp_cluster_name }}/auth/kubeconfig"
-    ignore_errors: true
-    loop:
-      - operator.yaml
-      - crds/heat.openstack.org_v1_heat_cr.yaml
-      - crds/heat.openstack.org_heats_crd.yaml
-      - service_account.yaml
-      - role_binding.yaml
-      - role.yaml
-
-  - name: delete heat-operator deployment if operator already deployed
+  - name: make sure mariadb pod is Running
     shell: |
-      if oc get deployments/heat-operator; then
-        oc delete deployments/heat-operator
-      fi
-      if oc get deployments/heat-api; then
-        oc delete deployments/heat-api
-      fi
-      if oc get deployments/heat-engine; then
-        oc delete deployments/heat-engine
-      fi
+      oc get pods mariadb -o json | jq .status.phase -r
+    register: mariadb_out
+    retries: 30
+    delay: 2
+    until: mariadb_out.stdout == "Running"
     args:
       chdir: "{{ base_path }}"
     environment:
       PATH: "/usr/local/bin:{{ ansible_env.PATH }}"
       KUBECONFIG: "{{base_path}}/dev-scripts/ocp/{{ ocp_cluster_name }}/auth/kubeconfig"
-    ignore_errors: true
 
-  - name: make sure mariadb pod is Running
+  #FIXME: this will soon be automated via heat-operator
+  - name: Add heat-operator service account to anyuid SCC
     shell: |
-      oc get pods mariadb -o json | jq .status.phase -r
-    register: mariadb_out
-    retries: 30
-    delay: 2
-    until: mariadb_out.stdout == "Running"
+      oc adm policy add-scc-to-user anyuid -n openstack -z heat-operator
```

Comment:
this was a temp work around. I see you've added it to the csv-merger as well now so I think it could be removed here.